Core/include/Acts/EventData/detail/BoundTrackParameters.hpp

```diff
@@ -51,120 +56,90 @@ class SingleBoundTrackParameters {
   /// an input here to be consistent with the other constructors below that
   /// that also take the charge as an input. The charge sign is only used in
   /// debug builds to check for consistency with the q/p parameter.
-  SingleBoundTrackParameters(std::shared_ptr<const Surface> surface,
-                             const ParametersVector& params, Scalar q,
-                             std::optional<CovarianceMatrix> cov = std::nullopt)
+  BoundTrackParameters(std::shared_ptr<const Surface> surface,
```

Comment:
Just as a thought: Should be default the particle hypothesis to pion? Then probably switch order with covariance... But not yet sure if that's a good idea...

Reply:
I also thought about that but decided against another default arg which is influencing a lot downstream